docs/helpers/Axes.md

```diff
@@ -37,6 +37,7 @@ Each axis - X, Y and Y2 should have the following properties:
 | ticks         | object   | null              | Refer [Ticks](Ticks.md)                                   |
 | rangeRounding | boolean  | true              | Toggle for range rounding                                 |
 | padDomain     | boolean  | true              | Toggle for disabling the padding (only for Y and Y2 axes) |
+| onLabelClick  | Function | undefined         | Any action that can be performed when clicking on label   |
```

Comment:
Nit: since you're updating the documentation, can you alphabetize these fields?

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/commits/f3cc18489de6df33f345b056dd0c815673f4c549#diff-e136c9760b04608293a9d079cc909772L35-R40)